pkgs/development/libraries/tracker/default.nix

```diff
@@ -46,7 +46,9 @@ stdenv.mkDerivation rec {
     pkg-config
   ];
 
+  strictDeps = true;
```

Comment:
Not sure what is the proper place but probably before first/after last dependency list, rather than in the middle. We currently [do not have it mapped](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md).

Reply:
Moved, thanks!